### DISCUSSION TITLE
Renewing Student Developer Pack

### DISCUSSION THREAD

**Discussion by ajeyer** [community/community#72812] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm attempting to renew my student developer pack, I got to the part to take a picture of my student ID or show a paper proving I'm currently enrolled. I'm an online student and have no ID. The site only has an option to take a picture, not upload a document, and after having spent 30+ minutes on waiting for the camera to open, the entire webpage closed out immediately after swapping to back camera. Now I have been unable to re-open the camera. Any tips/advice on getting proof I'm still enrolled incase my request gets denied since I was unable to submit a picture?

**Comment by Piero24** [comment]
@ajeyer  if you don't have a photo of the badge it's not a problem you can go to your school website on your personal profile and take a screenshot of the page where you see your profile photo and the name of the university as recommended by github itself.
